lib/components/App.js

```diff
@@ -89,7 +94,7 @@ export default class App extends React.Component {
             </p>
           </footer>
           <script>FilePizza()</script>
-          <ga.Initializer />
+          { process.env.DISABLE_GA != 'yes' ? <ga.Initializer /> : <div></div> }
```

Comment:
use `process.env.GA_ACCESS_TOKEN` as a predicate here, and change `<div></div>` to `null`